.github/workflows/test.yml

```diff
@@ -41,7 +41,14 @@ jobs:
             python-version: py35
             package: instrumentation
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Core Repo at SHA - ${{ github.sha }}
+        uses: actions/checkout@v2
+      - name: Checkout Contrib Repo @ SHA d4b5a955681400d038beb74b52aa74a08a4113bd
```

Comment:
Is there a way to move this SHA to a variable for reuse? I'm not super familar with GH actions, but if it doesn't support it, you could use YAML anchors

Reply:
I get why you would want that, I've add it as [an env var](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env) in the same folder. I think this solution looks clean and PRs can modify this reference if they want to prove their code is working.

We could even say the following:

To merge a PR on the Core Repo:

1. You must pass test on the Core Repo
2. You must get approvals on the Contrib Repo PR (if applicable)
3. The Core and Contrib repos will be merged at the same time

Reply:
Awesome!